packages/server/pipeStreamOverResponse.ts

```diff
@@ -8,11 +8,11 @@ const pipeStreamOverResponse = (
 ) => {
   res.onAborted(() => {
     readStream.destroy()
-    res.done = true
+    res.aborted = true
```

Comment:
rename wasn't necessary, but i did it so `aborted` means the same thing throughout the app